include/rcpputils/filesystem_helper.hpp

```diff
@@ -217,6 +218,26 @@ inline path remove_extension(const path & file_path, int n_times = 1)
   return new_path;
 }
 
+/**
+ * For a regular file p, returns the size determined as if by reading the st_size member of the
+ * structure obtained by stat.
+ * If p does not exist, reports an error.
+ *
+ * \param file_path Path to examine.
+ * \return The size of the file, in bytes.
+ */
+inline std::uintmax_t file_size(const path & file_path)
```

Comment:
Move implementation to cpp

Reply:
Just wondering if there's a reason `remove_extension` was inline but not this.
Is it b/c that one operated on or returned a path?